src/app/tests/suites/TestGlobalCommands.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Global Commands Tests
```

Comment:
Thinking of names like:

DelayTests, WaitTests (very focused)

ExecutionControl (a bit more broad, things like wait and maybe conditional ifs)

I would like to ensure that things that go together are somewhat related. I imagine "WaitForMs" and some fictional "factoryReset" or "AquireDifferentIPAddress" should likely belong in different buckets for readability sake.

Reply:
I renamed the cluster to `DelayCommands` and update the references accordingly. So this "simulated" cluster will host delay/timers related methods only.